python/ray/tests/test_actor_in_container.py

```diff
@@ -67,6 +67,40 @@ def run_pandas(self):
     ray.shutdown()
 
 
+@pytest.mark.skipif(sys.platform != "linux", reason="Only works on linux.")
+def test_actor_in_container_with_resource_limit():
```

Comment:
Can you add one more case to test that we can reuse the workers with the same resources string?

Reply:
I have added `test_reuse_worker_with_resource_limit`